src/components/side_panel/chart/treemap_chart/treemap_chart_design_panel.xml

```diff
@@ -0,0 +1,127 @@
+            t-call="o-spreadsheet-CategoryColorList"
+          />
+          <t t-else="" t-call="o-spreadsheet-SimpleColorScale"/>
+        </Section>
+      </t>
+    </SidePanelCollapsible>
+
+    <SidePanelCollapsible collapsedAtInit="false" title.translate="Headers and labels">
+      <t t-set-slot="content">
+        <Section title.translate="Headers" class="'pt-0 pb-0'">
+          <Checkbox
+            name="'showHeaders'"
+            label.translate="Show headers"
+            value="props.definition.showHeaders ?? defaults.showHeaders"
+            onChange="(showHeaders) => props.updateChart(this.props.figureId, { showHeaders })"
+          />
+        </Section>
+        <t t-if="showHeaders">
+          <TextStyler
+            class="'pt-0 o-header-style'"
+            updateStyle="(headerDesign) => props.updateChart(this.props.figureId, { headerDesign })"
+            style="props.definition.headerDesign || {}"
+            defaultStyle="defaults.headerDesign"
+            hasText="false"
+            hasBackgroundColor="true"
+          />
+        </t>
+
+        <Section title.translate="Labels" class="'pt-0 pb-0'">
+          <div class="d-flex flex-row gap-4">
+            <Checkbox
+              name="'showLabels'"
+              label.translate="Show labels"
+              value="props.definition.showLabels ?? defaults.showLabels"
+              onChange="(showLabels) => props.updateChart(this.props.figureId, { showLabels })"
+            />
+            <Checkbox
+              name="'showValues'"
+              label.translate="Show values"
+              value="props.definition.showValues ?? defaults.showValues"
+              onChange="(showValues) => props.updateChart(this.props.figureId, { showValues })"
+            />
+          </div>
+        </Section>
+        <t t-if="showValues || showLabels">
+          <TextStyler
+            class="'pt-0 o-values-style'"
+            updateStyle="(valuesDesign) => props.updateChart(this.props.figureId, { valuesDesign })"
+            style="props.definition.valuesDesign || {}"
+            defaultStyle="defaults.valuesDesign"
+            hasText="false"
+            hasVerticalAlign="true"
+          />
+        </t>
+      </t>
+    </SidePanelCollapsible>
+  </t>
+
+  <t t-name="o-spreadsheet-CategoryColorList">
+    <div class="mt-3">
+      <div class="o-fw-bold mb-2">Category</div>
+      <t t-foreach="getTreeGroupAndColors()" t-as="group" t-key="group.group">
+        <div class="d-flex align-items-center mb-2" t-att-data-id="group.group">
+          <RoundColorPicker
+            currentColor="group.color"
+            onColorPicked="(color) => this.setGroupColor(group.group, color)"
+          />
+          <span class="ps-2">
+            <span class="" t-esc="group.group"/>
+          </span>
+        </div>
+      </t>
+      <Checkbox
+        name="'highlightBigValues'"
+        label.translate="Highlight bigger values"
```

Comment:
I'm not sure about *"Highlight bigger values"*, you could argue it also highlights smaller values.
What about something like *"Use value-based gradient"*?